lib/utils/lifecycle.js

```diff
@@ -265,9 +265,11 @@ function runCmd_ (cmd, pkg, env, wd, stage, unsafe, uid, gid, cb_) {
   var shFlag = '-c'
 
   var customShell = npm.config.get('script-shell')
+  var customShellOpts = npm.config.get('script-shell-opts')
 
   if (customShell) {
     sh = customShell
+    shFlag += ' ' + customShellOpts
```

Comment:
shouldn't this just be `shFlag = customShellOpts`? It's conceivable that you wouldn't want `-c` used at all.